### PR TITLE
Documentation: New `README.md` file

### PR DESCRIPTION
**Pull request by Hywan**
Address https://github.com/hoaproject/Central/issues/19. This is a proposal for a new `README.md` format.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,22 +32,33 @@ manipulate promotion or special offer rules written by a user. To quote
...
+
+Considering the library has been installed with Composer, the following
+commands will run the test suites:
+
+```sh
+$ composer install
+$ vendor/bin/hoa test:run
+```
+
+For more information, please consult the [contributor
```
Comment (jubianchi) [inline README.md]: "consult" ? I don't know if it's correct... it seems a bit weird to me. We don't you just use "read" ?

File: README.md [hunk 2]
```diff
@@ -214,10 +235,46 @@ Have fun!
 
 ## Documentation
 
-Different documentations can be found on the website:
-[http://hoa-project.net/](http://hoa-project.net/).
+The [hack book of
+`Hoa\Ruler`](https://hoa-project.net/Literature/Hack/Ruler.html) contains
+detailed information about how to use this library and how it works.
```
Comment (jubianchi) [inline README.md]: I think you should write "contains detailed **on** about how to ..."

File: README.md [hunk 3]
```diff
@@ -22,22 +32,33 @@ manipulate promotion or special offer rules written by a user. To quote
...
+For more installation procedures, please read [the Source
+page](http://hoa-project.net/Source.html).
+
+## Testing
+
+Considering the library has been installed with Composer, the following
+commands will run the test suites:
+
+```sh
+$ composer install
```
Comment (Hywan) [inline README.md]: The point is “Composer”, not “installed”. If the library has not been installed with Composer, having this command makes no sens, but we can rephrase it.

File: README.md (hunk 1 above)
Comment (Hywan) [inline README.md]: This is correct but a little bit formal. I am rephrasing it.

File: README.md (hunk 2 above)
Comment (Hywan) [inline README.md]: “contained detailed on about”? Are you sure? I don't know this formulation.

Comment (Hywan) [inline README.md]: cc @dantleech

Comment (dantleech) [inline README.md]: What @Hywan makes perfect sense to me, I guess you meant `contains detailed information on how about to..` but still prefer what is currently there.

**Pull request merged**